packages/react/src/MeasureReportDisplay/MeasureReportGroup/MeasureReportGroup.tsx

```diff
@@ -0,0 +1,127 @@
+import { Box, Flex, Group, Paper, RingProgress, Text, Title } from '@mantine/core';
+import { formatCodeableConcept } from '@medplum/core';
+import { Measure, MeasureReportGroup } from '@medplum/fhirtypes';
+import React from 'react';
+import { QuantityDisplay } from '../../QuantityDisplay/QuantityDisplay';
+
+interface MeasureGroupProps {
+  readonly group: MeasureReportGroup;
+}
+
+interface MeasureProps {
+  readonly measure: Measure;
+}
+
+export function MeasureReportDisplayGroup(props: MeasureGroupProps): JSX.Element | null {
```

Comment:
For consistency purposes, let's use `{resourceType} + 'Display'`, so this would be called `MeasureReportGroupDisplay`.  The component, props, and filename should align.